agent/taskresource/envFiles/envfile.go

```diff
@@ -452,3 +469,58 @@ func (envfile *EnvironmentFileResource) UnmarshalJSON(b []byte) error {
 
 	return nil
 }
+
+// GetContainerName returns the container that this resource is created for
+func (envfile *EnvironmentFileResource) GetContainerName() string {
+	return envfile.containerName
+}
+
+// ReadEnvVarsFromEnvFiles reads the environment files that have been downloaded
+// and puts them into a list of maps
+func (envfile *EnvironmentFileResource) ReadEnvVarsFromEnvfiles() ([]map[string]string, error) {
+	var envVarsPerEnvfile []map[string]string
+
+	for _, envfilePath := range envfile.environmentFilesLocation {
```

Comment:
would be safer if using a getter with lock for environmentFilesLocation. otherwise it's implicitly relying on the fact that no one writes to it in between

Reply:
oh my bad, i forgot to update this part after adding lock to that var. will update, thanks for catching.